pos_anonymous_customer/models/res_partner.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    available_in_pos = fields.Boolean(
+        string='Available for POS',
```

Comment:
> The default is quite deliberately to not load the customer when not specifically set. 

I understalease, just add a comment to keep that future feature in the road mapnd, you're right, this is the expected behaviour. if not desired, a mass edit on res.partner, will easily realise the work.

About the more flexible approach. I'll add this feature, when i'll use this module. let's go accepting this PR as a first good step.

regards.